### ISSUE TITLE
Clear descriptions and demonstration of labeling different form fields

### ISSUE THREAD

**Issue by yatil** [w3c/wai-tutorials#109]
**Original Comment:**

> The labeling page is misleading, I believe. it reads:
> "Provide labels to identify all form controls including text fields, checkboxes, radio buttons, drop-down menus, and button elements. In most cases this is done by using the <label> element, and in rare cases the title attribute."
> 
> So, do button elements use the label element or the title attribute? This paragraph is oversimplifying things and will make some people think that they need to use one of these for <button>.
> 
> Maybe there should be a table for HTML native form controls that shows how the item is typically labelled, and that would show people that inputs of type submit and button elements are different and how radio buttons are usually in need of grouping.
